z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -249,7 +249,24 @@ $ openstack subnet set --no-dns-nameservers <use_network_subnet>
 $ openstack subnet set --dns-nameserver 198.51.100.86 <use_network_subnet>
 ```
 
-For the multiple Openshift Container Platform, user can create multiple subnets or tenant network to deploy it. 
+
+10. Support to deploy multiple OCPs with multiple networks
+
+Now we support the following scenarios for multiple ocps :
+- Deploy multiple OCPs on ZVM multi-vlan network and the detail steps you can refer ICIC doc [link](https://www.ibm.com/docs/en/cic/1.2.0?topic=mfvc-zvm-multiple-vlan-networks-pass-through-different-zvm-vswitchesthe-first-time-configuration) to configure it.
+
```

Comment:
`and the detail steps you can refer ICIC doc [link](https://www.ibm.com/docs/en/cic/1.2.0?topic=mfvc-zvm-multiple-vlan-networks-pass-through-different-zvm-vswitchesthe-first-time-configuration) to configure it.`
--->
You can refer ICIC doc to [create mulitiple vlan networks](https://www.ibm.com/docs/en/cic/1.2.0?topic=mfvc-zvm-multiple-vlan-networks-pass-through-different-zvm-vswitchesthe-first-time-configuration) , then choose one network and set the network in the related inventory.yaml file section.

The belows same.

Reply:
[link] should be the link content.